pkg/controller/applicationlayer/applicationlayer_controller_test.go

```diff
@@ -41,6 +40,8 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	ctrlrfake "github.com/tigera/operator/pkg/ctrlruntime/client/fake"
```

Comment:
If you're tweaking imports, please put this one with the other tigera/operator imports.